client/app/message/inbox.html

```diff
@@ -37,8 +37,10 @@ <h1 class="section__title">
                                     <span am-time-ago="message.sentDate | amUtc | amLocal"></span>
                                 </td>
                                 <td>
-                                    <i class="hot-ds-icon-trash-bin pointer"
-                                       ng-click="inboxCtrl.confirmDeleteMessage(message.messageId)"></i>
+                                    <button type="button ng-binding" class="button"
```

Comment:
Are you sure of value of type here?
It looks weird to me to have ng-binding here.
Shouldn't this be in `class` instead?

Reply:
By the way, is it useful?

Reply:
Good spot! Fix in https://github.com/hotosm/tasking-manager/pull/767